blurb/blurb.py

```diff
@@ -467,7 +467,7 @@ def finish_entry():
             if not body:
                 throw("Blurb 'body' text must not be empty!")
             text = textwrap_body(body)
-            for naughty_prefix in ("- ", "Issue #", "bpo-"):
+            for naughty_prefix in ("- ", "Issue #", "bpo-", "gh-"):
```

Comment:
Maybe this should be case-insensitive.  `GH-*` is often spelled uppercase.

Reply:
This is now case-insensitive, and I added the test for it.